client/src/pages/Projects/index.tsx

```diff
@@ -1,24 +1,124 @@
 import Page from "src/layouts/page";
-import { sampleData } from "./projectData";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faTwitter,
+  faDiscord,
+  faTelegram,
+  faMedium,
+} from "@fortawesome/free-brands-svg-icons";
+import { Themes } from "src/entities/common.entities";
+import {
+  sampleData,
+  ProjectData,
+  ProjectLogos,
+  ProjectURLs,
+} from "src/entities/project.entities";
 
 import { RootState } from "src/store";
 import { useSelector } from "react-redux";
 
-function Projects() {
+const Projects = () => {
```

Comment:
This looks like a component. you can put it in the `/component` folder. Try to keep `page` clean from components definition